app/forms/steps/client/contact_details_form.rb

```diff
@@ -30,6 +30,10 @@ def choices
 
       private
 
+      def string_choices
+        choices.map { |choice| choice.value.to_s }
```

Comment:
You can do if you prefer `choices.map(&:to_s)` because `to_s` is delegated to `value` in the `ValueObject` superclass 😜 it is just a shortcut anyways.

Reply:
Nice I'll do that - didn't think of the delegation!